tests/unit/htdocs/modules/system/SystemMaintenanceTest.php

```
@@ -0,0 +1,269 @@
+        $this->setProtectedProperty($obj, 'prefix', 'xoops_');
+
+        return $obj;
+    }
+
+    /**
+     * Stub SHOW TABLES to return a known table list for validation.
+     *
+     * @param \XoopsMySQLDatabase|\PHPUnit\Framework\MockObject\MockObject $db
+     * @param array $tables  Unprefixed table names
+     */
+    private function stubShowTables($db, array $tables): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)